scripts/release_mp.py

```diff
@@ -292,11 +292,13 @@ def upload(self, url, marketplace, dryrun=False, owner=None):
         """ Upload the given Marketplace package and update the config file."""
         uploaded = [url + ":"]
         mp_to_upload = self.mp_config.get(marketplace, "mp_to_upload")
-        for pkg in glob.glob(mp_to_upload):
-            if os.path.isfile(pkg):
-                retcode = self.upload_file(url, pkg, dryrun=dryrun, owner=owner)
-                if retcode == 0:
-                    uploaded.append(os.path.realpath(pkg))
+        mp_to_upload = mp_to_upload.replace(" ", "")
```

Comment:
Why? Could you add a test to highlight the need to strip? 
If needed, why replace versus strip?
https://www.programiz.com/python-programming/methods/string/strip

Reply:
We need to `replace` because we could have a mix of coma + space or just coma:
```
*/target/a-*.zip, */*/target/b-*.zip,*/target/c-*.zip
```
This is only to be less strict and allow both syntaxes.

Reply:
We can remove that line and use `strip()` in the next-next line:
```python
for pkg in glob.glob(mp.stip()):
```